src/test/java/com/mongodb/jdbc/MongoResultSetTest.java

```diff
@@ -229,14 +229,24 @@
         try {
             mongoStatement = new MongoStatement(mongoConnection, "test", true);
         } catch (SQLException e) {
-            e.printStackTrace();
```

Comment:
Hmm odd, `throw new RuntimeException(e)` is definitely better. I wonder why we had that.